code/modules/virus2/effect.dm

```diff
@@ -928,3 +931,200 @@ var/list/compatible_mobs = list(/mob/living/carbon/human, /mob/living/carbon/mon
 
 		to_chat(mob, "Suddenly, your knowledge of languages comes back to you.")
 	..()
+
+
+/datum/disease2/effect/lubefoot
+	name = "Self-lubricating Footstep Syndrome"
+	stage = 3
+
+/datum/disease2/effect/lubefoot/activate(var/mob/living/carbon/mob,var/multiplier)
```

Comment:
the shoes are meant to be a holdover. Only  way to disable the honk is to remove its feet